Source/Core/DolphinQt2/Settings/GeneralPane.cpp

```diff
@@ -38,6 +38,8 @@ void GeneralPane::CreateLayout()
   CreateAnalytics();
 #endif
   CreateAdvanced();
+
+  m_main_layout->setMargin(0);
```

Comment:
It still squishes the horizontal margins, yeah, though I don't know if it's a problem or not. Not a blocker for me, at least.